easybuild/tools/github.py

```diff
@@ -1088,3 +1087,26 @@ def validate_github_token(token, github_user):
         _log.info("GitHub token can be used for authenticated GitHub access, validation passed")
 
     return sanity_check and token_test
+
+
+def find_easybuild_eb():
```

Comment:
@Caylo please add a unit test for this (just check whether it returns a path an existing easyconfig for EasyBuild)